api-reference/beta/api/grouplifecyclepolicy_renewgroup.md

```diff
@@ -0,0 +1,71 @@
+# groupLifecyclePolicy: renewgroup
```

Comment:
Is this on groupLifeCyclePolicy or on groups/{id}?  Seems weird that I renew a group using the policy, which might apply to multiple groups?